### PR TITLE
Add table packages tabu and tabularx

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

**Comment by pirapira** [comment]
Is `tabu` and `tabularx` supposed to be used together?  Isn't it enough to add one of them only?

**Comment by jamesray1** [comment]
I had read the CTAN info.

https://ctan.org/pkg/tabu

I think you can just add tabu and not need to also use tabularx.

> The tabu en­vi­ron­ment may be used in place of tab­u­lar, tab­u­lar* and tab­u­larx en­vi­ron­ments, as well as the ar­ray en­vi­ron­ment in maths mode.

**Comment by jamesray1** [comment]
But you can also add both, and I wasn't sure.

File: Paper.tex [hunk 1]
```diff
@@ -22,7 +22,15 @@
 \usepackage[bookmarks=true, unicode=true, pdftitle={Ethereum Yellow Paper: a formal specification of Ethereum, a programmable blockchain}, pdfauthor={Gavin Wood and others as per https://github.com/ethereum/yellowpaper/commits/master},pdfkeywords={Ethereum, Yellow Paper, blockchain, virtual machine, cryptography, decentralised, singleton, transaction, generalised},pdfborder={0 0 0.5 [1 3]}]{hyperref}
 %,pagebackref=true
 
-% I will insert other table packages in here after this is merged.
+%\usepackage{longtable}
+%https://stackoverflow.com/questions/1918366/automated-line-breaks-in-latex-tables/1918465#1918465
+%tabularx seems better, but we can include this if we need tables to break across pages automatically, as well as tabu and longtabu
+
+%\usepackage{tabularx}
+%https://stackoverflow.com/a/1919447/7438857
```
Comment (pirapira) [inline Paper.tex]: I think the above comments are not necessary.

**Comment by jamesray1** [comment]
Revised.

**Pull request merged**